src/workos.ts

```diff
@@ -1,5 +1,4 @@
 import axios, { AxiosError, AxiosInstance, AxiosResponse } from 'axios';
-import { version } from '../package.json';
```

Comment:
We're not importing from `package.json` (which is outside of the `src` directory) so that we can flatten the output directory.